### PR TITLE
Remove the padding around sections on homepage

### PR DESCRIPTION
**Pull request by jfrankl**
This is the first step in cleaning up the site design on desktop.

image

### CODE REVIEW

**Comment by jwalgran** [comment]
There are are two broken tests which broke because they were looking for certain classes. Are you comfortable fixing the tests to match the revised markup, or would you like an assist?

```
======================================================================
FAIL: test_online_trained_user_content (apps.home.tests.HomeTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/app/apps/home/tests.py", line 238, in test_online_trained_user_content
    response.assert_your_contributions_visible(True)
  File "/opt/app/apps/home/tests.py", line 203, in assert_your_contributions_visible
    '<section class="your-contributions', it_is)
  File "/opt/app/apps/home/tests.py", line 192, in _assertContains
    self.test.assertContains(self.response, text, count=count)
  File "/usr/local/lib/python2.7/dist-packages/django/test/testcases.py", line 360, in assertContains
    " (expected %d)" % (real_count, text_repr, count))
AssertionError: Found 0 instances of '<section class="your-contributions' in response (expected 1)

======================================================================
FAIL: test_public_homepage_content (apps.home.tests.HomeTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/app/apps/home/tests.py", line 223, in test_public_homepage_content
    response.assert_about_visible(True)
  File "/opt/app/apps/home/tests.py", line 199, in assert_about_visible
    '<section class="section-about section-home">', it_is)
  File "/opt/app/apps/home/tests.py", line 192, in _assertContains
    self.test.assertContains(self.response, text, count=count)
  File "/usr/local/lib/python2.7/dist-packages/django/test/testcases.py", line 360, in assertContains
    " (expected %d)" % (real_count, text_repr, count))
AssertionError: Found 0 instances of '<section class="section-about section-home">' in response (expected 1)

----------------------------------------------------------------------

**Comment by jfrankl** [comment]
Could use an assist on that.

On Thu, Apr 2, 2015 at 11:34 AM, Justin Walgran <notifications@github.com>
wrote:

> There are are two broken tests which broke because they were looking for
> certain classes. Are you comfortable fixing the tests to match the revised
> markup, or would you like an assist?
>
> ======================================================================
> FAIL: test_online_trained_user_content (apps.home.tests.HomeTestCase)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/opt/app/apps/home/tests.py", line 238, in test_online_trained_user_content
>     response.assert_your_contributions_visible(True)
>   File "/opt/app/apps/home/tests.py", line 203, in assert_your_contributions_visible
>     '<section class="your-contributions', it_is)
>   File "/opt/app/apps/home/tests.py", line 192, in _assertContains
>     self.test.assertContains(self.response, text, count=count)
>   File "/usr/local/lib/python2.7/dist-packages/django/test/testcases.py", line 360, in assertContains
>     " (expected %d)" % (real_count, text_repr, count))
> AssertionError: Found 0 instances of '<section class="your-contributions' in response (expected 1)
>
> ======================================================================
> FAIL: test_public_homepage_content (apps.home.tests.HomeTestCase)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/opt/app/apps/home/tests.py", line 223, in test_public_homepage_content
>     response.assert_about_visible(True)
>   File "/opt/app/apps/home/tests.py", line 199, in assert_about_visible
>     '<section class="section-about section-home">', it_is)
>   File "/opt/app/apps/home/tests.py", line 192, in _assertContains
>     self.test.assertContains(self.response, text, count=count)
>   File "/usr/local/lib/python2.7/dist-packages/django/test/testcases.py", line 360, in assertContains
>     " (expected %d)" % (real_count, text_repr, count))
> AssertionError: Found 0 instances of '<section class="section-about section-home">' in response (expected 1)
>
> ----------------------------------------------------------------------
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/nyc-trees/pull/1036#issuecomment-88949174>.
>



-- 
Jeff Frankl, UI/UX Designer

Azavea  |  340 N 12th St, Suite 402, Philadelphia, PA
Web azavea.com  |  Blog azavea.com/blogs  |  Twitter @azavea
<http://twitter.com>

**Comment by jwalgran** [comment]
I did a before/after comparison in side by side tabs and confirmed the improvements. I'll fix up the assertions in the tests to use the updated classes.

File: src/nyc_trees/apps/home/tests.py [hunk 1]
```diff
@@ -222,17 +222,17 @@ def test_public_homepage_content(self):
         response = self._render_homepage()
         response.assert_about_visible(True)
         response.assert_training_visible(False)
-        response.assert_your_contributions_visible(False)
+        response.assert_contributions_visible(True)
```
Comment (jwalgran) [inline src/nyc_trees/apps/home/tests.py]: Contributions should be visible on the logged out homepage

**Comment by jwalgran** [comment]
@jfrankl I fixed the tests. Are you cool to resolve the conflict and merge this?

**Comment by jwalgran** [comment]
Merging because the markup changes landed in ``develop`` and we need to get the test fixes in there too.

**Pull request merged**